tests/aio-create.sh

```diff
@@ -201,6 +201,10 @@ pushd /opt/openstack-ansible
     # Pin flask so it stops breaking xenial and other versions
     echo "Flask==0.12.2" >> /opt/openstack-ansible/global-requirement-pins.txt
     enable_ironic
+  
+  elif [ "${RE_JOB_SCENARIO}" == "queens" ]; then
+    git checkout "stable/queens"  # Branch checkout of Pike (Current Stable)
```

Comment:
The comment says `Pike`, probably a copy-pasta error?

Reply:
Done